src/frontends/tensorflow/src/op_table.cpp

```diff
@@ -221,6 +221,7 @@ const std::map<std::string, CreatorFunction> get_supported_ops() {
         {"BatchMatMul", CreatorFunction(translate_batch_mat_mul_op)},
         {"BatchMatMulV2", CreatorFunction(translate_batch_mat_mul_op)},
         {"BatchMatMulV3", CreatorFunction(translate_batch_mat_mul_with_type_op)},
+        {"BatchMatrixInverse", CreatorFunction(translate_batch_matrix_inverse_op)},
```

Comment:
please use `translate_matrix_inverse_op` that has been already implemented. Just expand `translate_matrix_inverse_op` with new op `BatchMatrixInverse` in `default_op_check`. Please remove other code you implemented. We just re-use existing code.
Please remove the test as well, no chance to verify it with new versions of TF, unfortunately:(

Really sorry for this.

Btw, you may do contribution to TF repository by updating docs for `BatchMatrixInverse` to mention that it is deprecated and recommend `MatrixInverse`